src/DoubleTheDonation/FormExtension/Field.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace GiveDoubleTheDonation\DoubleTheDonation\FormExtension;
+
+use Give\Framework\FieldsAPI\Field as FieldsApiField;
+
+/**
+ * @unreleased
+ */
+class Field extends FieldsApiField
+{
+    const TYPE = 'dtd';
+
+    protected $label;
+
+    /**
+     * @unreleased
+     */
+    public function setLabel(?string $label)
+    {
+        $this->label = $label;
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getLabel(): ?string
+    {
+        return $this->label;
+    }
```

Comment:
You can just use the `Concerns\HasLabel;` trait for these.

Reply:
Good idea!

Reply:
Resolved: https://github.com/impress-org/give-double-the-donation/pull/17/commits/6c2603dff604602560feef28fd8c70ac824135e1